scripts/linux_arm64_librdkafka_install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+TEMP=/tmp/librdkafka
+LIB_PATH=/usr/local/lib
+LIB_RDKAFKA=librdkafka.so
+VERSION=v1.8.2
+
+# Check if we have /usr/local/lib/librdkafka.so
+if [ -f "$LIB_PATH/$LIB_RDKAFKA" ]; then
+    echo "librdkafka is already installed"
+else
+    echo "Installing librdkafka..."
+    # Build librdkafka
+    git clone -b $VERSION https://github.com/edenhill/librdkafka.git $TEMP
+    cd /tmp/librdkafka
+    ./configure
+    make
+    make install
+    cd ${PWD}
+
+    # Cleanup
+    rm -rf $TEMP
+fi
+
+# Check if we have the env varibale set
+if ! tr ':' '\n' <<< "$LD_LIBRARY_PATH" | grep -xFq "$LIB_PATH"; then
+    echo 'LD_LIBRARY_PATH is not set. Please set it with export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib'
```

Comment:
Tiny nit: The `:` is only needed if there are things already in `LD_LIBRARY_PATH`. That's why the other stuff uses this construct: `${PKG_CONFIG_PATH:+:$PKG_CONFIG_PATH}`.

That uses variable expansion to do some magic. Basically, if `PKG_CONFIG_PATH` isn't empty (the `:+` part), return `:` then `$PKG_CONFIG_PATH`. But if it's empty, it still just returns empty. Where this is being used, the entry is being added to the front of the variable, so that's why the `:` comes first. In this case, you could do `${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH:}$LIB_PATH`.

```suggestion
    echo "LD_LIBRARY_PATH is not set. Please set it with export LD_LIBRARY_PATH=\${LD_LIBRARY_PATH:+\$LD_LIBRARY_PATH:}$LIB_PATH"
```

Reply:
That makes sense, added the change.